src/blocks/formItemText/save.tsx

```diff
@@ -0,0 +1,24 @@
+import { useBlockProps } from '@wordpress/block-editor';
+
+export const save = (props) => {
+	const { attributes } = props;
+	const blockProps = useBlockProps.save({
+		className: 'wp-block-custom-form-item',
+	});
+
+	return (
+		<div {...blockProps}>
+			<label>
+				<span>{attributes.label}</span>
+				{attributes.require && (
+					<span>*</span>
+				)}
+			</label>
+			<input type='text' name={attributes.id}
+				placeholder={attributes.placeholder}
+				required={attributes.require}
+				className='wp-block-custom-form-item__text'
+			/>
```

Comment:
一般的に、propsの属性ごとに改行する形が良いかと思います！
```suggestion
			<input
			    type='text'
			    name={attributes.id}
				placeholder={attributes.placeholder}
				required={attributes.require}
				className='wp-block-custom-form-item__text'
			/>
```

Reply:
ご確認いただきありがとうございます！
修正しました！
https://github.com/amiS-acat/work-re-step-cuscom-block/pull/1/commits/77e369834220b382f35a408548d1ee200211a0c1